src/legate.mk

```diff
@@ -86,6 +86,14 @@ endif
 NVCC_FLAGS += -g
 endif
 
+ifeq ($(strip $(DEBUG)),1)
```

Comment:
Note that cunumeric sets `DEBUG` (and thus `DEBUG_LEGATE`) independently of legate.core. We will have to be careful not to have `DEBUG_LEGATE` control things like struct layout.